### DISCUSSION TITLE
Multi-Platform client

### DISCUSSION THREAD

**Discussion by mcredstoner** [lanis-mobile/lanis#294] [Ideas]
My feature idea would be support for platforms like Linux, Windows or macOS.

**Comment by alessioc42** [comment]
Hi there,
While [Flutter](https://docs.flutter.dev/reference/supported-platforms) and our code are compatible with the listed platforms, we do not plan to add desktop support. This is due to 2 factors:
1. Code complexity, maintainability and maintenance: Adding support for more platforms would cause a lot of headaches just due to the unnecessary overhead of managing the platform projects. Adding iOS was already quite difficult. 
2. It would not be the purpose of the app. I started with Lanis-**Mobile** because I wanted a better client on my phone that would send me notifications, a feature the website simply cannot support. On desktop I would always choose the website because it is feature complete. If you still want a desktop client, check out @ObnoxiousOliver's [BetterLANiS](https://github.com/ObnoxiousOliver/BetterLANiS), which wraps the school portal in an interesting interface. (It's been a while since I tested this project, it might be broken by now)
